crates/trycp_server/src/main.rs

```diff
@@ -245,6 +252,25 @@ fn get_info_as_json() -> String {
     format!("{{{}}}", result)
 }
 
+/// very dangerous, runs whatever strings come in from the internet directly in bash
```

Comment:
lol - trycp only runs on servers that are only internally vpc accessible though, right?

Reply:
Yeh and the only way to call this function is through the other pub functions, I just kept zippys comment so that no-one makes this pub